Mathlib/Tactic/Linter/AttributeInstanceIn.lean

```diff
@@ -0,0 +1,71 @@
+/-
+Copyright (c) 2024 Michael Rothgang. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Michael Rothgang, Damiano Testa
+-/
+
+import Lean.Elab.Command
```

Comment:
```suggestion
import Lean.Elab.Command
import Lean.Linter.Util
```
I do not know if you were tricked by `Shake`: this import is needed for `logLint`.

Reply:
I was not tricked by shake - I removed this import manually and realised later that it wouldn't build...